resources/dicts/thoughts/dead/starclan/general.json

```
@@ -223,12 +247,61 @@
             "newborn"
         ]
     },
+    {
+        "id": "dead_parent_to_alive_child",
+        "thoughts":[
+            "Snuck into r_c's dream to comfort {PRONOUN/r_c/object} after a hard day",
+            "Can't stop fretting over r_c even though there's nothing {PRONOUN/m_c/subject} can do",
+            "Is talking the ears off any cat willing to listen to {PRONOUN/m_c/object} go on and on and on about r_c",
+            "Is feeling guilty watching r_c miss {PRONOUN/m_c/object}",
+            "Deeply regrets leaving r_c without {PRONOUN/m_c/object}",
+            "Visited r_c's dream to tell {PRONOUN/r_c/object} just how immensely proud {PRONOUN/m_c/subject} {VERB/m_c/are/is} of {PRONOUN/r_c/subject}"
+        ],
+        "random_living_status": [
+            "starclan",
+            "darkforest",
+            "unknownresidence"
+        ],
+        "relationship_constraint": [
+            "parent/child"
+        ],
+        "main_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "dead_outsider_parent_to_alive_child",
+        "thoughts":[
+            "Regrets not joining a Clan if it meant being able to raise r_c",
+            "Still doesn't trust Clan cats but is happy r_c is safe",
+            "Is just glad r_c was saved from meeting the same fate {PRONOUN/m_c/subject} did",
+            "Is so greatful to c_n for saving r_c"
+        ],
+        "random_backstory_constraint": [
+            "orphaned1",
+            "orphaned2",
+            "orphaned4",
+            "orphaned5"
+            ],
+        "main_living_status": [
+            "unknownresidence"
+        ],
+        "relationship_constraint": [
+            "parent/child"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
     {
         "id": "dead_child_to_alive_child_sibling",
         "thoughts":[
             "Wants to let r_c know {PRONOUN/m_c/subject} {VERB/m_c/are/is} okay",
             "Wishes {PRONOUN/m_c/subject} could play with r_c",
-            "Is watching r_c play"
+            "Is watching r_c play",
+            "Snuck into r_c's dreams to play with {PRONOUN/r_c/object} one last time",
+            "Visits r_c in {PRONOUN/r_c/poss} to tell them to live for the both of them",
```

```suggestion
            "Visits r_c in {PRONOUN/r_c/poss} dreams to tell them to live for the both of them",
```
again, could these dream focused ones be moved to STAR and DREAM skill constraints?